### ISSUE TITLE
Support for Connect API?

### ISSUE THREAD

**Issue by mla** [lukec/stripe-perl#203]
Hey there. Hope everyone is well.

Just wanted to confirm that there is no support yet for the Connect-related API. Is that right? For example, creating an Account along with requested_capabilities and such:
https://stripe.com/docs/connect/custom-accounts

I didn't see it mentioned in docs, and grepping the sources for the endpoints didn't surface anything.

Thanks.

**Comment by sherrardb** [comment]
I don't think that we support the Connect API. i believe that this is my first time reading about it.

i'll leave this open in case it is something i am able to double back to later.

and of course you are welcome to submit a PR with code and unit tests, if you end up hacking something together for your needs.
